index.html

```diff
@@ -181,9 +181,17 @@
       .email-helptip {
         display: inline-block;
       }
-      .email-helptip:hover .helptip-content {
+      .email-helptip:hover .helptip-content, .email-helptip-clicked .helptip-content {
         display: block;
       }
+      .email-helptip .helptip-content:after {
```

Comment:
Is this working without specifying a `display` rule? I thought the `:before` and `:after` psuedo-classes needed both `display` and `content` defined, but I could be mistaken.

Reply:
It is working; I basically grabbed this code from [this CodePen](https://codepen.io/Konrud/pen/ygaYqg).